.github/workflows/build.yml

```diff
@@ -57,20 +57,18 @@ jobs:
         run: |
           echo IMAGE_REPOSITORY=$(echo ${{ github.repository }} | tr '[:upper:]' '[:lower:]') >> $GITHUB_ENV
           echo IMAGE_TAG=$(echo ${{ github.ref }} | tr '[:upper:]' '[:lower:]' |  awk '{sub(/([^\/]*\/){2}/,""); gsub(/\/|_/, "-")}1') >> $GITHUB_ENV
-          echo HAS_SECRET_PAT=$(echo ${{ env.HAS_SECRET_PAT }})
       - name: Login to GitHub Container Registry
         uses: docker/login-action@v1
-        if: env.HAS_SECRET_PAT == 'true'
```

Comment:
Login is only required, when push should be done.

Reply:
fixed with 18754c98189c58cefea2e83203bfa1360f581eb8